### ISSUE TITLE
Internal Server Error when merging Findings

### ISSUE THREAD

**Issue by 37b** [DefectDojo/django-DefectDojo#4586]
https://github.com/DefectDojo/django-DefectDojo/blob/9e88e0480c9afd58b21354d7de38ea6f12fd0f2c/dojo/forms.py#L574

```
File "./dojo/forms.py", line 575, in __init__
--
self.fields.keyOrder = ['finding_to_merge_into', 'findings_to_merge', 'append_description', 'add_endpoints', 'append_reference']
AttributeError: 'dict' object has no attribute 'keyOrder'```

**Comment by valentijnscholten** [comment]
Should be changed into `field_order` due to django 3:https://docs.djangoproject.com/en/3.0/ref/forms/api/#notes-on-field-ordering

**Comment by valentijnscholten** [comment]
Are you able to do a PR and test it?

**Comment by 37b** [comment]
Yep I can do that

**Comment by valentijnscholten** [comment]
fixed in dev / 2.3.0
